apps/nextjs/src/app/dashboard/organizations/[org_slug]/create/_components/phases-info.tsx

```
@@ -0,0 +1,303 @@
+        header: "pb-4",
+      }}
+    >
+      {formData.phases.length > 0 && (
+        <CardContent className="space-y-4">
+          {formData.phases.map((phase, index) => (
+            <PhaseDetailsNestedCard
+              phase={phase}
+              index={index}
+              formData={formData}
+              setFormData={setFormData}
+            />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)